src/PodActions.jsx

```diff
@@ -77,72 +78,98 @@ const PodDeleteModal = ({ con, pod }) => {
 export const PodActions = ({ con, onAddNotification, pod, isPodService }) => {
     const Dialogs = useDialogs();
 
+    const startPod = () => {
+        if (isPodService) {
+            systemctl_spawn(["start", pod.Labels.PODMAN_SYSTEMD_UNIT], con.uid === 0).catch(ex => {
+                const error = cockpit.format(_("Failed to start pod $0"), pod.Name);
+                onAddNotification({ type: 'danger', error, errorDetail: ex.message });
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2286-9d63ad0b-20251017-133332-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)